src/backend/views/layouts/main.njk

```diff
@@ -63,12 +63,26 @@ height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
   {% include "../partials/cookieBanner.njk" %}
 {% endblock %}
 
+{% block main %}
```

Comment:
This is well-structured and organised code.

Some minor comments:

- While this isn't a functional issue, it's a good idea to be consistent with the use of single or double quotation marks throughout your templates, line 23.
- Comments: Adding comments to your code can help other developers understand what each section is intended to do. This is especially useful in complex templates (I'm not sure if this is or isn't complex though 🙂  ).
- Error Handling: Consider adding error handling in case a necessary piece (like the included partials) doesn't exist or fails to load for some reason.

Reply:
To solve the quotation formatting, we can add Prettier's plugin `prettier-plugin-nunjucks` to the prettier config

Reply:
- add double quotes
- agree about leaving comments but only if it's really needed / complex
- will look into catching template errors
- not sure about  `prettier-plugin-nunjucks` as it still a working progress module, will see if theres another one we can use.